meta-lmp-bsp/recipes-kernel/linux/linux-ti-staging/am64xx-sk/0001-linux-ti-staging-add-wireguard-config-to-kernel-buil.patch

```diff
@@ -0,0 +1,36 @@
+From d009a2b3296230c00f1a008908f739a1caed6d18 Mon Sep 17 00:00:00 2001
+From: Tim Anderson <tim.anderson@foundries.io>
+Date: Thu, 18 Feb 2021 13:39:57 -0700
+Subject: [PATCH] linux-ti-staging: add wireguard config to kernel build
+
+Signed-off-by: Tim Anderson <tim.anderson@foundries.io>
+---
+ ti_config_fragments/v8_defconfig_map.txt | 2 +-
+ ti_config_fragments/wireguard.cfg        | 4 ++++
+ 2 files changed, 5 insertions(+), 1 deletion(-)
+ create mode 100644 ti_config_fragments/wireguard.cfg
+
+diff --git a/ti_config_fragments/v8_defconfig_map.txt b/ti_config_fragments/v8_defconfig_map.txt
+index 00a1ae124..6044909c2 100644
+--- a/ti_config_fragments/v8_defconfig_map.txt
++++ b/ti_config_fragments/v8_defconfig_map.txt
+@@ -1,5 +1,5 @@
+ # Release Defconfigs
+-classification: SDK_Release_Defconfigs type: ti_sdk_arm64_release defconfig: defconfig config_file: None extra_configs: arm64_prune.cfg v8_baseport.cfg v8_ipc.cfg connectivity.cfg audio_display.cfg multimedia.cfg k3_soc.cfg systemd.cfg
++classification: SDK_Release_Defconfigs type: ti_sdk_arm64_release defconfig: defconfig config_file: None extra_configs: arm64_prune.cfg v8_baseport.cfg v8_ipc.cfg connectivity.cfg audio_display.cfg multimedia.cfg k3_soc.cfg systemd.cfg wireguard.cfg
+ # Debug Defconfigs
+ classification: SDK_Debug_Defconfigs type: ti_sdk_arm64_debug defconfig: defconfig config_file: None extra_configs: arm64_prune.cfg v8_baseport.cfg v8_ipc.cfg connectivity.cfg audio_display.cfg multimedia.cfg k3_soc.cfg systemd.cfg debug_options.cfg
+ # RT Defconfigs
+diff --git a/ti_config_fragments/wireguard.cfg b/ti_config_fragments/wireguard.cfg
+new file mode 100644
+index 000000000..23b489bd3
+--- /dev/null
++++ b/ti_config_fragments/wireguard.cfg
+@@ -0,0 +1,4 @@
++CONFIG_NET_UDP_TUNNEL=m
++CONFIG_TIPC=m
++CONFIG_TIPC_MEDIA_UDP=y
++CONFIG_TIPC_DIAG=m
```

Comment:
So this is not for enabling wireguard, but for supporting it from the kernel perspective.

As you're trying to use the linux bsp recipe, can you see if you can upstream this fragment changes (probably extend the networking ones) will be good with it?

I'm just concerned that using linux-ti-staging might also not be enough (e.g. missing systemd/docker options), so we might get to a point of replicating the recipe as we do for the other boards (rpi, nxp, etc), which is a custom lmp-recipe (which uses our fragments), but leveraging the kernel source tree provided by the bsp vendor.